<launch>
     <node pkg="openpose_ros" name="openpose_ros" type="openpose_ros.py" output="screen">
          <param name="image_topic" value="/zed2i/zed_node/rgb/image_rect_color"/>
          <param name="depth_topic" value="/zed2i/zed_node/depth/depth_registered"/>
          <param name="camera_info_topic" value="/zed2i/zed_node/rgb/camera_info"/>
     </node>
     <!-- Display -->
     <node pkg="rviz" type="rviz" name="rviz" args="-d $(find openpose_ros)/rviz/openpose_ros.rviz" />

</launch>